examples/real_data.ipynb

```diff
@@ -0,0 +1,769 @@
+    "    plt.plot(catalog_in[\"e1\"], catalog_in[\"e2\"], ',')\n",
+    "    plt.xlabel(\"e1\")\n",
+    "    plt.ylabel(\"e2\")\n",
+    "\n",
+    "#make_plots(data_2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e71d6d00-f18e-4e27-969b-f02f13032713",
+   "metadata": {},
+   "source": [
+    "<a id=\"Running_CLMM\"></a>\n",
+    "## 5. Running CLMM on the dataset\n",
+    "We use the functions similar to `examples/Paper_v1.0/gt_and_use_case.ipynb`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1c4b465d-9e1c-4b63-a43e-eeea6139b462",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from clmm import Cosmology\n",
+    "cosmo = Cosmology(H0=70.0, Omega_dm0=0.27-0.045, Omega_b0=0.045, Omega_k0=0.0)\n",
+    "\n",
+    "# We consider MaxBCG J140.53188+03.76632\n",
+    "cluster_z = 0.2701 # Cluster redshift\n",
+    "cluster_ra = 140.54565 # Cluster Ra in deg\n",
+    "cluster_dec = 3.77820 # Cluster Dec in deg\n",
+    "\n",
+    "obs_galaxies = data_2\n",
+    " \n",
+    "obs_galaxies = obs_galaxies[(obs_galaxies['z']>(cluster_z+0.1))]\n",
+    "\n",
+    "# Area cut: the query is made for the whole field and this can simplify the processing.\n",
+    "select = obs_galaxies['ra'] < cluster_ra + 12./60./np.cos(cluster_dec/180.*np.pi)\n",
+    "select &= obs_galaxies['ra'] > cluster_ra - 12./60./np.cos(cluster_dec/180.*np.pi)\n",
+    "select &= obs_galaxies['dec'] < cluster_dec + 12./60.\n",
+    "select &= obs_galaxies['dec'] > cluster_dec - 12./60.\n",
+    "obs_galaxies = obs_galaxies[select]\n",
+    "\n",
+    "obs_galaxies['id'] = np.arange(len(obs_galaxies))\n",
+    "\n",
+    "# Put galaxy values on arrays.\n",
+    "gal_ra = obs_galaxies['ra'] # Galaxies Ra in deg\n",
+    "gal_dec = obs_galaxies['dec'] # Galaxies Dec in deg\n",
+    "gal_e1 = obs_galaxies['e1'] # Galaxies elipticipy 1\n",
+    "gal_e2 = obs_galaxies['e2'] # Galaxies elipticipy 2\n",
+    "gal_z = obs_galaxies['z'] # Galaxies observed redshift\n",
+    "gal_id = obs_galaxies['id'] # Galaxies ID"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c1f44ef-ee3b-490b-8d54-4d433b50a93f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shear profile\n",
+    "# Using/testing clmm.dataops functions (details are in the notebook for Paper V1.0).\n",
+    "# This step is for testing the CLMM functionalities; we will not use the result in later step.\n",
+    "\n",
+    "import clmm.dataops as da\n",
+    "# Convert elipticities into shears.\n",
+    "gal_ang_dist, gal_gt, gal_gx = da.compute_tangential_and_cross_components(cluster_ra, cluster_dec,\n",
+    "                                                                          gal_ra, gal_dec,\n",
+    "                                                                          gal_e1, gal_e2,\n",
+    "                                                                          geometry=\"flat\")\n",
```

Comment:
Please, remove `geometry='flat'`. The default is now to use the correct `curve` geometry, using astropy functionality.

Reply:
Thanks. I have removed it.